lookout/core/metrics/values.py

```diff
@@ -0,0 +1,129 @@
+import os
+from threading import Lock
+
+from prometheus_client.mmap_dict import mmap_key, MmapedDict
+
+"""This file is a copy of
+  https://github.com/prometheus/client_python/blob/master/prometheus_client/values.py
+  modified to increment the variables using the Kahan sum algorithm to maintain precision even
+  when aggregating long sequences of numbers."""
+
+
+class MutexValue(object):
```

Comment:
Too complex, all these classes. I want a single class with everything. Put all the implementation inside `ConfidentCounter`

Reply:
If you still want the MutexValue inside ConfidentCounter I can do it, but rn it does no longer make sense to me.